content/yaml-code-signing/code-signing-identities.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Code signing identities
+description: How to set up code signing identities
+weight: 4
+aliases: /code-signing-yaml/code-signing-identities
+---
+
+Teams can use code signing identities to manage their code signing files in **Team settings > codemagic.yaml settings > Code signing identities** and reference them from their `codemagic.yaml` configuration.
+
+## Managing and uploading files
+
+Team owner permissions are required to upload and edit files under the **Code signing identities** section. However, all team members can view the file info for any of the uploaded files.
+
+### Apple certificates
+
+Codemagic lets you upload certificates with the `.p12` or `.pem` extension. When uploading, Codemagic will ask you to provide the certificate password along with a unique **Reference name**, which can then be used in the `codemagic.yaml` configuration to fetch the specific file.
```

Comment:
certificate password is required only when the certificate is password-protected